src/main/io/servo_sbus.c

```diff
@@ -61,9 +63,17 @@ bool sbusServoInitialize(void)
         return false;
     }
 
-    servoSbusPort = openSerialPort(portConfig->identifier, FUNCTION_SERVO_SERIAL, NULL, NULL, SERVO_SBUS_UART_BAUD, MODE_TX, SERVO_SBUS_OPTIONS);
-    if (!servoSbusPort) {
-        return false;
+    bool servoserialCheckRxPortShared(const serialPortConfig_t *portConfig)
+    {
+        return portConfig->functionMask & FUNCTION_RX_SERIAL;
```

Comment:
@tech2c I'd like to add it do 5.0 Could you please extend this check to verify if RX protocol is SBUS, not only if port is RX assigned?